test/hotspot/gtest/gtestMain.cpp

```diff
@@ -208,6 +216,14 @@ static char** remove_test_runner_arguments(int* argcp, char **argv) {
   return new_argv;
 }
 
+// This is generally run once for a set of tests, but if we have vm_assert, or other_vm
+// tests, a new process is forked and this will be called, for each of those tests.
```

Comment:
is the comma needed?

Reply:
Which one? :) I wanted to convey that the fork and execution of `runUnitTestsInner` is done for each test. I will reword and restructure.

Reply:
The second one after "called" tickled my comma sense. But I am not a native speaker and may be wrong.